### ISSUE TITLE
Ability to track a whole directory

### ISSUE THREAD

**Issue by nyurik** [andsens/homeshick#220]
the `track` command seem to only support individual files. Is there a good way to track a whole directory?

**Comment by andsens** [comment]
Yup. If you check out [the linking table](https://github.com/andsens/homeshick/wiki/Symlinking#linking-table) you'll see that symlinks are symlinked. So if you add a directory to your repo outside of `home/` and symlink to that from inside `home/` homeshick will link to that symlink in your actual `$HOME`, meaning you have an entire directory in your castle that is tracked and also accessible from a place of your choosing in `$HOME`.

**Comment by nyurik** [comment]
@andsens if i understood you correctly, homeshick will handle symlinked directories properly, but it does not offer any tools to **create** such symlinks from existing directories?  E.g. if nothing is tracked, there is no homeshick command to add it, but i can do it myself with some hand-crafted `ln -s ...` command?

**Comment by nyurik** [comment]
@andsens thanks for the great in-depth reply!   Do you think it would make sense to add dir handling to the `track` command (or a new `track-dir` command?)
